src/core/lib/gpr/log.cc

```diff
@@ -30,10 +30,17 @@
 
 GPR_GLOBAL_CONFIG_DEFINE_STRING(grpc_verbosity, "ERROR",
                                 "Default gRPC logging verbosity")
+GPR_GLOBAL_CONFIG_DEFINE_STRING(grpc_stacktrace_minloglevel, "",
+                                "Messages logged at the same or higher level "
+                                "than this will print stacktrace")
+
+static constexpr gpr_atm GPR_LOG_SEVERITY_UNSET = 0xAB;
```

Comment:
That feels a bit weird using `0xAB` and `0xCD`

Reply:
Okay I changed it to more non-random constants.